src/commands/hawku.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const config = require('../../config');
+const hawkuConfigFilePath = config.commands.hawkuPath
+const convert = require('xml-js');
+
+const options = {compact: true, ignoreDeclaration: true, ignoreAttributes: true, nativeTypeAttributes: true};
+
+// Ran every time this command is ran, in-case the user changes their configuration
+// This loads the entire file into a JS object after converting it
+function loadConfigFile() {
+    let hawkuUserSettings = require('fs').readFileSync(hawkuConfigFilePath, 'utf8');
+    hawkuUserSettings = convert.xml2js(hawkuUserSettings, options);
+    hawkuUserSettings = hawkuUserSettings.Configuration;
+    return hawkuUserSettings;
+}
```

Comment:
Unsure if I care that this isn't async. It's probably fine.
(Suggestion relies on the import change I suggested above.)

```suggestion
const loadConfigFile = () => {
  return xml2js(
    fs.readFileSync(hawkuConfigFilePath, 'utf8'), options
  ).Configuration;
}
```